test/integration/builder/updates.js

```diff
@@ -339,5 +339,36 @@ module.exports = function(knex) {
           );
         });
     });
+
+    describe('#3553 update with bigint', function() {
+      if (typeof BigInt === 'undefined') return;
+
+      it('should allow insert with BigInt', async function() {
+        const firstName = '#3553';
+        await knex('accounts').insert([{ balance: 5, first_name: firstName }]);
+        await knex('accounts').update([
+          { balance: BigInt(Number.MAX_SAFE_INTEGER) },
```

Comment:
`Number.MAX_SAFE_INTEGER` is still a regular integer, correct?  If so, then you might want to use a larger number here.

Reply:
Oh, and of course: read the number back out of the database as well.  (Just to make sure it didn't get truncated or anything like that)